mapswipe_workers/tests/unittests/test_firebase_connection.py

```diff
@@ -0,0 +1,20 @@
+import unittest
+
+from mapswipe_workers import auth
+
+
+class TestFirebaseConnection(unittest.TestCase):
+    def setUp(self):
+        self.fb_db = auth.firebaseDB()
+
+    def tearDown(self):
+        self.fb_db = None
+
+    def test_create_project(self):
```

Comment:
we should rename this function. Will this return something? Can we test, assert for a specific error in firebase? e.g. no connection or something like this?

e.g. defined here: https://firebase.google.com/docs/reference/admin/python/firebase_admin.exceptions

Reply:
Good idea. I created an issue for this: #408